software/kutt.yml

```diff
@@ -1,15 +1,14 @@
 name: Kutt
 website_url: https://kutt.it
 source_code_url: https://github.com/thedevs-network/kutt
-description: A modern URL shortener with support for custom domains and custom URLs.
+description: Modern URL shortener with support for custom domains and custom URLs.
 licenses:
   - MIT
 platforms:
   - Nodejs
   - Docker
 tags:
   - URL Shorteners
-demo_url: https://kutt.it
```

Comment:
Demo links which are duplicates of the main website link are allowed if the main website is actually an instance of the program. This allows quickly differentiating projects which provide an interactive demo, from those that do not.